doc/system_requirements.md

```diff
@@ -138,3 +134,40 @@ Covers:
 * [feat~row-level-security~1](#row-level-security)
 
 Needs: dsn
+
+### Row Level Security with Tenants 
+
+#### Tables With Tenants Restrictions
+`req~tables-with-tenants-restrictions~1`
+
+Data Owners can define for each row in the table if it belongs to only one user (tenant).
+
+Covers:
+
+* [feat~row-level-security~1](#row-level-security)
+
+Needs: dsn
+
+#### Tables with Both Roles and Tenants Restrictions
+`req~tables-with-both-roles-and-tenants-restrictions~1`
+
+If a table contains both roles restrictions and tenants restrictions, then only tenants restrictions will be applied. 
```

Comment:
This is not what should happen. Better to be more restrictive and apply both at the same time:
1. You need to be the right tenant **and**
1. You need the right role
to see a dataset.

Reply:
fixed